src/isolate.rs

```diff
@@ -579,3 +592,112 @@ impl DerefMut for OwnedIsolate {
     unsafe { self.cxx_isolate.as_mut() }
   }
 }
+
+#[repr(C)]
+/// Instances of this struct can be passed to isolate.get_heap_statistics to get heap statistics from V8. 
+pub struct HeapStatistics{
+  total_heap_size_:usize,
+  total_heap_size_executable_:usize,
+  total_physical_size_:usize,
+  total_available_size_:usize,
+  total_global_handles_size_:usize,
+  used_global_handles_size_:usize,
+  used_heap_size_:usize,
+  heap_size_limit_:usize,
+  malloced_memory_:usize,
+  external_memory_:usize,
+  peak_malloced_memory_:usize,
+  number_of_native_contexts_:usize,
+  number_of_detached_contexts_:usize,
+  does_zap_garbage_:usize,
+}
+
+impl HeapStatistics {
+  
+  // Number of bytes V8 has allocated for the heap. This can grow if usedHeap needs more.
```

Comment:
Can you use triple slash comments `///` so cargo doc picks these doc comments up.